kpops/manifests/argo.py

```
@@ -0,0 +1,50 @@
+import enum
+from typing import Any
+
+from typing_extensions import override
+
+
+class ArgoEnricher:
+    @property
+    def key(self) -> str:
+        return NotImplemented
```

I believe it should be `raise` instead of `return`